src/core_atmosphere/Registry.xml

```diff
@@ -1135,6 +1135,72 @@
                         <var_array name="lbc_scalars" />
 
                 </stream>
+		<stream name="da_state"
+                        type="input;output"
+                        filename_template="mpasout.$Y-$M-$D_$h.$m.$s.nc"
+                        input_interval="initial_only"
+                        output_interval="0_06:00:00"
+                        immutable="true">
```

Comment:
Since the "da_state" stream is not conditionally compiled into the model, I think we should attach the "jedi_da" package to the stream. You can check the "lbc_in" stream for an example of how to attach a package to a stream.

Reply:
Yes I will. FMI What are the semantics of adding a "packages" attribute to a stream?

Reply:
Done.

Reply:
A stream with packages will only be read or written (effectively, it will only be active) if at least one of the attached packages is active. So in this case, even though the "da_state" stream has an `input_interval` and and `output_interval`, it will only actually be read or written if `config_jedi_da` is `true` in the namelist (which activates the "jedi_da" package.

Reply:
I think you need to remove the `>` character from the line with the `immutable` attribute.

Reply:
Right, done. My bad for not running the nit tests.